CadRevealFbxProvider.Tests/BatchUtils/GeometryInstancerTests.cs

```diff
@@ -273,27 +263,14 @@ private void CheckNodePattern1(CadRevealNode node)
     private void CheckNodePattern2(CadRevealNode node)
     {
         // Check that the non-instanced objects are still all distinct
-        for (var i = 1; i < node.Geometries.Length; i++)
+        foreach (int j in new[] { 0, 1, 2, 5, 6 })
```

Comment:
Consider commenting the meaning of these magic numbers, maybe create an array and add comments behind each number